### ISSUE TITLE
Bug: "does not pass the 'from' option to 'postcss.parse'"

### ISSUE THREAD

**Issue by hugo-t-b** [GoogleChromeLabs/postcss-jit-props#21]
Vite 3 shows the following warning when this plugin is enabled:

```
A PostCSS plugin did not pass the `from` option to `postcss.parse`. This may cause imported assets to be incorrectly transformed. If you've recently added a PostCSS plugin that raised this warning, please contact the package author to fix the issue.
```

The docs for `postcss.parse` are [here](https://postcss.org/api/#postcss-parse).

**Comment by argyleink** [comment]
strange, this plugin doesn't call `postcss.parse()`, so there's no parameter to pass to make the warning go away

**Comment by kelvindecosta** [comment]
Hey @argyleink 

I think this might be the offending line: 

https://github.com/GoogleChromeLabs/postcss-jit-props/blob/82b9f12d056f53844eb798a2b1e20ab10baae850/index.js#L62

PS: Happy New Year!

**Comment by argyleink** [comment]
fixed in v1.0.9 👍🏻

**Comment by fkrauthan** [comment]
For some reason I get this error with version 1.0.9 too: My postcss config looks like this:

```
const postcssJitProps = require('postcss-jit-props');
const postcssCustomMedia = require('postcss-custom-media');
const postcssGlobalData = require('@csstools/postcss-global-data');
const autoprefixer = require('autoprefixer');
const OpenProps = require('open-props');

module.exports = {
  plugins: [
    postcssJitProps(OpenProps),
    postcssGlobalData({
      files: [
        require.resolve('open-props/media.min.css'),
      ],
    }),
    postcssCustomMedia(),
    autoprefixer,
  ]
}
```
